packages/amplify_authenticator/lib/src/widgets/authenticator_banner.dart

```diff
@@ -79,4 +120,15 @@ extension on StatusType {
         return Icons.error;
     }
   }
+
+  ColorSwatch<int> get color {
+    switch (this) {
+      case StatusType.info:
+        return Colors.blue;
+      case StatusType.error:
+        return Colors.red;
+      default:
+        return Colors.blue;
```

Comment:
So, warning & success will be blue by default? Seems like yellow/green may be better choices for these, no?

Reply:
@Equartey you can completely remove `StatusType.warning` & and `StatusType.success` from the codebase since they are unused. Then the default case can be removed.